opendebates/static/less/base.less

```diff
@@ -397,7 +619,7 @@ body {
 }
 
 .modal-header .close:hover {
-  background: @row-bg;
+  // background: @row-bg;
```

Comment:
yeah this is all the original stylesheet that i'm working over time to consume into new styles. There is a comment that designates that old section and before launch i expect everything under it to be removed.